### ISSUE TITLE
[MJAVADOC-608] Unable to run javadoc:aggregate on multi-module projects with one or more pom modules when using Java modules (jigsaw).

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-javadoc-plugin#895]
**[Anders Hanson](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hanson)** opened **[MJAVADOC-608](https://issues.apache.org/jira/browse/MJAVADOC-608?redirect=false)** and commented

The following error is displayed and build fail when runnin mvn javadoc:aggregate on a multi-module project where one or more child modules are of pom type (\<packaging>pom\</packaging>) when using maven modules (jigsaw)
 

```
[ERROR] Creating an aggregated report for both named and unnamed modules is not possible.
 [ERROR] Ensure that every module has a module descriptor or is a jar with a MANIFEST.MF containing an Automatic-Module-Name.
```

I've attached a minimal project showing the problem. Run either mvn javadoc:aggregate or mvn site:site

maven  module with packaging pom should be excluded from the unnames module lest, they are not java modules and will not produce any jar files, named or not. 

This is with maven 3.6.0 and jdk 11.0.3 (latest AdoptJDK) 

---

**Affects:** 3.1.0

**Attachments:**
- [module-test.zip](https://issues.apache.org/jira/secure/attachment/12971504/module-test.zip) (_3.53 kB_)

1 votes, 3 watchers

**Comment by jira-importer** [comment]
**[Jessica-Aileen Alten](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jiai64)** commented

Same problem here - and there seems to be no chance to skip over this pom type module (perhaps another bug: MJAVADOC-613 ).

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Issue confirmed. Not sure how difficult this issue is, but we can guide you fixing it.

**Comment by jira-importer** [comment]
**[Jessica-Aileen Alten](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=jiai64)** commented

With maven-javadoc-plugin version 3.1.1 it seems to work for me now! :)

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Fix as part of another issue in 3.1.1
